### DISCUSSION TITLE
Bombs are too bouncy

### DISCUSSION THREAD

**Discussion by Alasdairbugs** [SuperTux/supertux#2054] [Ideas]
Bombs are not made of rubber. Try upthrowing a bomb against a wall, or hay, it will bounce off with exactly the same momentum incident to it, and it looks stupid, feels stupid, and makes using bombs hard and annoying.

Making bombs have more weight will make them more useable in dynamic gameplay.

**Comment by CPa129** [comment]
Also, in version 0.3.0, 0.3.1 and 0.3.2 SVN made Tux-carryable objects heavy.
